### ISSUE TITLE
Error Message Misleading running `geneis new` when secret-provider is not set 

### ISSUE THREAD

**Issue by xiujiao** [genesis-community/genesis#354]
when the secret-provider is not set or maybe the one is set is not found, running `genesis new` has the following error message:
```
genesis new drgao-aws-proto
[installing genesis runtime]
Setting up new environment drgao-aws-proto...
[ERROR] Safe target for http://127.0.0.1:8203 not found.  Please run

  #C{safe target <name> "http://127.0.0.1:8203"

then authenticate against it using the correct auth method before
re-attempting this command.
```
`safe targets` shows `http://127.0.0.1:8203` is aliased as `dev` and `safe get secret/handshake` works. A better Error message would suggest running `genesis secret-provider` to set your safe target after verifying `safe targets` list the target we want to use.

Steps to reproduce
Run `genesis init` and `genesis new` for a deployment, then pushed the deployment repo to the github. Then on a different jumpbox, git pull the deployment repo, run `genesis new` with a local safe target

**Comment by jhunt** [comment]
I think you got a hold of 2.6.17, which is actually 2.7.0-rc in disguise.  We should talk about this some time, @dennisjbell

**Comment by xiujiao** [comment]
> I think you got a hold of 2.6.17, which is actually 2.7.0-rc in disguise. We should talk about this some time, @dennisjbell

yeah, you are right. I do have genesis 2.6.17. I thought this is a good one to use since it says the latest release while there is another one says pre-release for 2.7.0

**Comment by jhunt** [comment]
Yeah, we had a pipeline mixup. ¯\_(ツ)_/¯

**Comment by dennisjbell** [comment]
Can we confirm that this was a transient problem, and close this issue?  If still happening, lets see if its resolved in the v2.7.0-rc3 pre-release.
